### DISCUSSION TITLE
Manifest URLs between Versions differ

### DISCUSSION THREAD

**Discussion by ElCbo** [ScoopInstaller/Scoop#5576] [Help and Questions]
Hi Folks,

I'm trying to write a manifest for a tool with different URLs between versions. So I got this 
`{
    "version": "1.0",
    "url": "https://abc.com/tool/1-{unknown-hash}-tool-v1.0.7z",
    "extract_dir": "tool",
    "bin": [
      "tool.exe"
    ]
}
`
but after an update I got this
`{
    "version": "1.1",
    "url": "https://abc.com/tool/0-{another-hash}-tool-v1.1.7z",
    "extract_dir": "tool",
    "bin": [
      "tool.exe"
    ]
}
`
There is no rule for generating the hash prior downloading the file. Can scoop handle this, so that I can explicitly install tool@1.0, even when 1.1 is in the bucket.

best regards

**Comment by HUMORCE** [comment]
**Captured variables** will NOT update when install specify version. instead, all **Version variables** works.

https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate#internal-substitutable-variables

**Comment by ElCbo** [comment]
I don't quite understand, do you have an example?

**Reply by HUMORCE** [reply]
see https://github.com/ScoopInstaller/Scoop/issues/5573
